core/src/main/java/com/linecorp/armeria/common/multipart/AggregatedBodyPart.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.multipart;
+
+import javax.annotation.Nullable;
+
+import com.google.common.base.MoreObjects;
+
+import com.linecorp.armeria.common.HttpData;
+
+/**
+ * A complete body part whose headers and content are readily available.
+ */
+public final class AggregatedBodyPart {
```

Comment:
- Could introduce some common interface between `AggregatedBodyPart` and `AggregatedHttpMessage` for accessing the content?
- Could add some static factory methods, like `AggregatedHttp{Request,Response}` got?
- Could make this class an interface?